app/services/ui_notifications/rss_notifications_checker.rb

```diff
@@ -70,5 +72,19 @@ def rss_notification_blueprint
     def notification_already_exists?(item)
       !!Notification.unscoped.find_by_message(item.summary)
     end
+
+    def load_rss_feed
+      uri = URI.parse(@url)
+      http = Net::HTTP.new(uri.host, uri.port)
```

Comment:
If you use this instead of the ```Net::HTTP.start(uri.host, uri.port) do |http|``` form, does it correctly close the connection after this function or will it wait for the garbage collection?

Reply:
it should close the connection once you call the http.request.body

Reply:
I don't think `http.request.body` closes the connection, but it looks like HTTP#request [runs](http://ruby-doc.org/stdlib-1.9.3/libdoc/net/http/rdoc/Net/HTTP.html) the request in a start using the block. Though I agree, using start with block would be more readable and ruby-like.